release/resources/stdlib_modules.json

```diff
@@ -401,7 +401,8 @@
                 "module-ballerinax-java.jdbc",
                 "module-ballerinax-mssql",
                 "module-ballerinax-mysql",
-                "module-ballerinax-postgresql"
+                "module-ballerinax-postgresql",
```

Comment:
Same

Reply:
removed in https://github.com/ballerina-platform/ballerina-standard-library/pull/1929/commits/3d92859879abd09244544e8666289860f78170e5